### DISCUSSION TITLE
The thing of Copy-Pasting ChatGPT Solutions on Codewars

### DISCUSSION THREAD

**Discussion by patriciasbar** [codewars/codewars.com#3252] [General]
Just kicked off my Codewars Python journey and noticed that many solutions seem to be direct copies of suggestions from ChatGPT. How do I know this? After submitting my code, I often consult GPT for feedback on best practices and to ensure my code is written in a Pythonic way. I use it for feedback, not for coding.

What do you all think about this trend of copying and pasting code from GPT?

**Comment by MurariAmbati** [comment]
I think this trend is stupid because ChatGPT can only go so far in giving solutions, and tends to give wrong answers as it is not a capable advanced AI yet. It cannot give solutions to very difficult level problems.

**Reply by patriciasbar** [reply]
agreed! even with ChatGPT Plus, I find myself correcting its Python coding suggestions once in a while

**Comment by hobovsky** [comment]
Debating on what one thinks about using ChatGPT to solve kata is not really relevant, because submitting AI-generated code is forbidden  by [Codewars Code of Conduct](https://docs.codewars.com/community/code-of-conduct/#behavior-that-is-not-tolerated) (the **"Cheating and plagiarising"** point). At the same time, it's quite difficult to tell whether a solution is actually generated or not. It happens that someone posts a comment like _"This user is a cheater and this solution was generated by ChatGPT"_ under a solution which was submitted before even the ChatGPT existed.
